new_wp.sh

```diff
@@ -4,82 +4,98 @@
 VP_NONE='\033[00m'
 VP_RED='\033[01;31m'
 VP_GREEN='\033[01;32m'
-VP_YELLOW='\033[01;33m'
+#VP_YELLOW='\033[01;33m'
```

Comment:
- Just FYI: There are even more colors for 265-color terminals: https://commons.wikimedia.org/wiki/File:Xterm_256color_chart.svg
- If you set run this script on different operating systems, `\33[` escape codes might not work on certain OS's. The following notation should always work: `ink_red="\\x1b[38;5;204m"`